frame/babe/src/tests.rs

```diff
@@ -126,6 +128,99 @@ fn first_block_epoch_zero_start() {
 	})
 }
 
+fn make_vrf_output(
+	slot_number: u64,
+	pair: &sp_consensus_babe::AuthorityPair
+) -> (VRFOutput, VRFProof, [u8; 32]) {
+	let pair = sp_core::sr25519::Pair::from_ref(pair).as_ref();
+	let transcript = sp_consensus_babe::make_transcript(&Babe::randomness(), slot_number, 0);
+	let vrf_inout = pair.vrf_sign(transcript);
+	let vrf_randomness: sp_consensus_vrf::schnorrkel::Randomness = vrf_inout.0
+		.make_bytes::<[u8; 32]>(&sp_consensus_babe::BABE_VRF_INOUT_CONTEXT);
+	let vrf_output = VRFOutput(vrf_inout.0.to_output());
+	let vrf_proof = VRFProof(vrf_inout.1);
+
+	(vrf_output, vrf_proof, vrf_randomness)
+}
+
+#[test]
```

Comment:
Most of the duplication in the tests below could be factored out, but won't block merge on this.